content/applications/sales/point_of_sale/shop/barcode.rst

```diff
@@ -58,5 +58,5 @@ Log employees
 -------------
 
 You can also use a barcode scanner to log your employees. To do so, :ref:`restrict access
-<restrict-employee-pos>` to the POS and :ref:`use barcodes to log your employees in
-<employee-barcode>` your POS.
+<employee_login/configuration>` to the POS and :ref:`use barcodes to log your employees in
+<employee_login/badge>` your POS.
```

Comment:
```suggestion
<employee_login/configuration>` to the POS and :ref:`assign badge IDs to your employees
<employee_login/badge>` for logging into into your POS.
```

Reply:
This change shouldn't be made in 16.0, but on previous versions (since this PR targets 16.0 and not for this page, I can only make the minimum change to avoid conflict)